feat: UTC-288: Add bulk import predictions api to sdk

hakan458 opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17779739596)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/595
<!-- Follow Merge -->

Adding ImportPredictionsAPI to SDK

Commit:
feat: UTC-288: Add bulk import predictions api to sdk

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17779678097

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17779739596

Commit:
add sdk test

jombooth commented on label_studio/data_import/api.py:
```diff
@@ -415,7 +415,48 @@ def create(self, request, *args, **kwargs):
+@method_decorator(
+    name='post',
+    decorator=extend_schema(
+        tags=['Import'],
```
maybe `['Import', 'Predictions']`? but honestly I don't know what impact this field has

hakan458 replied:
Also not sure if adding the additional tag will change anything. Definitely wont change where it lives in the navigation

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio-enterprise/actions/runs/17781164619
